examples/02_decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,70 @@
+"""
+Decoding with decoder metaestimator: face vs house recognition
+==============================================================
+
+Here is a simple example of decoding with the decoder metaestimator,
+reproducing a face vs house discrimination task on the study Haxby 2001.
+
+    * J.V. Haxby et al. "Distributed and Overlapping Representations of Faces
+      and Objects in Ventral Temporal Cortex", Science vol 293 (2001),
+      p 2425.-2430.
+
+"""
+
+# Load Haxby dataset
+from nilearn.datasets import fetch_haxby
+data_files = fetch_haxby(n_subjects=1)
```

Comment:
Don't specify the number of subjects: by default it's only 1, and the second one.